src/features/MoviesBrowser/PopularPeople/index.js

```diff
@@ -1,19 +1,32 @@
-import { useState } from "react";
+import { useQuery } from "react-query";
 import PeopleList from "../../../common/PeopleList";
-import { usePopularPeopleData } from "./usePopularPeopleData";
+import { API_KEY, ApiPopularPeople } from "../../../codesAPI";
+import Loading from "../../../common/Loading";
+import Error from "../../../common/Error";
 
 const PopularPeople = () => {
-  const [currentPage, setCurrentPage] = useState(1);
-  const popularPeopleData = usePopularPeopleData(currentPage);
+  const { data, isLoading, isError, error } = useQuery(
+    "popularPeople",
+    async () => {
+      const response = await fetch(`${ApiPopularPeople}?api_key=${API_KEY}`);
+      if (!response.ok) {
+        throw new Error("Network response was not ok");
+      }
+      return response.json();
+    }
+  );
```

Comment:
Great Idea to use react-query here 😎
I believe the old usePopularPeopleData.js file is not necessary anymore, so make sure to remove it 👌